templates/contest/list.html

```diff
@@ -124,7 +124,7 @@
 {% macro contest_join(contest, request) %}
     {% if not request.in_contest %}
         <td>
-            {% if request.profile in contest.organizers.all() %}
+            {% if request.profile in contest.editor_ids or request.profile in contest.tester_ids %}
```

Comment:
Does this actually work? I would have expected `request.profile.id in ...`.

Reply:
Should be `request.profile.id`, my bad.